run_iwyu_tests.py

```diff
@@ -107,6 +107,7 @@ def setUp(self):
       'prefix_header_includes_add.cc': prefix_headers,
       'prefix_header_includes_keep.cc': prefix_headers,
       'prefix_header_includes_remove.cc': prefix_headers,
+      'template_default_args_roundtrip.cc': ['-std=c++11'],
```

Comment:
This test doesn't really require c++11, does it? Might as well just leave the `-std` switch at its default.